### PR TITLE
feat: separate federal and state lands overlays

### LINKED ISSUE #38: State Land Overlay

### LINKED ISSUE #39: Federal Land Overlay

### PR DESCRIPTION
**Pull request by kmal808**
Separating the federal-state overlay into federal-lands & state-lands. Both new data files have been simplified to 10% and had the precision reduced to 0.00001. 

Closes ❌ #38 & #39 

See ⏬ for feature preview 👀 

https://user-images.githubusercontent.com/20919083/208237957-d917ba61-72c1-49bc-a955-0f4eddfd7fbf.mov

### CODE REVIEW

**Comment by tyliec** [comment]
Nice job, props for also including a video 👍

File: scripts/map.js [hunk 1]
```diff
@@ -468,7 +471,35 @@ var loadFederalState = function () {
...
+//* state land overlay
+var loadState = function () {
+	$.getJSON('./data/state-land.min.geojson', function (geojson) {
+		var stripes = new L.StripePattern({
+			height: 2,
+			width: 2,
+			weight: 1,
+			spaceWeight: 1,
+			angle: 30,
+			color: '#5cc649',
```
Comment (tyliec) [inline scripts/map.js]:
```suggestion
			color: '#b349c6',
```
Screenshot 2022-12-17 at 6 28 52 PM

Thinking about it some more, we should probably color these differently. Randomly picking a color based off what a random website told me to

https://colorpicker.me/#b349c6

**Review by tyliec — APPROVED** [review]
1 comment about the color, but otherwise LGTM 👍

File: scripts/map.js (hunk 1 above)
Comment (kmal808) [reply scripts/map.js]: Suggestion committed ➡️  [1df5d29](https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas/pull/47/commits/1df5d294e45ea3139179443f68cb4981fb05ae16)

**Pull request merged**